packages/components/CHANGELOG.md

```diff
@@ -24,6 +24,7 @@
 -   `Button`: Convert to TypeScript ([#46997](https://github.com/WordPress/gutenberg/pull/46997)).
 -   `QueryControls`: Convert to TypeScript ([#46721](https://github.com/WordPress/gutenberg/pull/46721)).
 -   `Notice`: refactor to TypeScript ([47118](https://github.com/WordPress/gutenberg/pull/47118)).
+-   Lock the `__experimentalPopoverPositionToPlacement` function and rename it to `popoverLegacyPositionToPlacement` ([#47505](https://github.com/WordPress/gutenberg/pull/47505)).
```

Comment:
I think we ended up deciding in https://github.com/WordPress/gutenberg/pull/47229 that it's nice to keep the `__experimental` prefix so that can quickly identify which APIs are non-public.

Or maybe we could use a new `__private` prefix for these cases?

I saw @youknowriad discussing these sorts of things in Slack. What do you think?

When the dust settles on 6.2 we should update `backward-compatibility.md` with the best practices that we land on.

Reply:
Oh, but this is a function! not a prop. OK then I guess that's fine. Maybe. I don't know anymore 😭

Reply:
Added the `__experimental` prefix for good measure (https://github.com/WordPress/gutenberg/pull/47505/commits/9f3158b79c741d6aa29d989a7f8994f89f3dafea). As you say, we can align naming conventions after 6.2 is released